doc/man/zcash-cli.1

```diff
@@ -3,7 +3,7 @@
 .SH NAME
 zcash-cli \- manual page for zcash-cli v1.0.7
 .SH DESCRIPTION
-Zcash RPC client version v1.0.7
+Zcash RPC client version v1.0.7\-24a24abf7\-dirty
```

Comment:
that's strange, there's a commit right before the manpages commit that contains all the version string changes